lib/maintenance_smelt.pm

```diff
@@ -64,4 +64,13 @@ sub get_packagebins_in_modules {
     return map { $_->{name} => $_ } @arr;
 }
 
+sub is_embargo_update {
+    my ($incident, $type) = @_;
+    return 0 if ($type =~ /PTF/);
+    my $url = "https://smelt.suse.de/api/v1/basic/incidents/$incident/";
+    my $res      = Mojo::UserAgent->new->get($url)->result;
+    die 'Request to ' . $url . ' failed, response code ' . $res->code if $res->code > 299;
+    return defined($res->json->{embargo});
+}
+
```

Comment:
@michaelgrifalconi among the things to do for the bot, this is one of the datapoints that the bot should do, as the tests should not query smelt for this.

Reply:
> @michaelgrifalconi among the things to do for the bot, this is one of the datapoints that the bot should do, as the tests should not query smelt for this.

several comments : 
- yes bot suppose to do this and actually is doing it already see ( https://github.com/openSUSE/qem-bot/pull/127 , https://github.com/openSUSE/qem-bot/pull/128 , https://github.com/openSUSE/qem-dashboard/pull/338) 
-  BUT seems like we still have some bug in the flow https://progress.opensuse.org/issues/153334
- NOT agree that we should remove it even after we will be sure that qem-bot do the job right . I see this like "second line of defense "

Reply:
Just a comment, hoping to be not wrong: 
the `get($url)` is equivalent to `curl`, unless you suggest to use _perl_ commands only;
moreover about the `return defined(...)` I'm afraid that it is always defined:
*not-embargo* has the string `null`, *embargo* has some `date-time`: both defined.

Reply:
About the `return defined(...)`, I'm afraid that it is always defined:
not-embargo has the string null, embargo has some date-time: both defined.
I hope to be not wrong.

Reply:
>  I see this like "second line of defense "

it is no line of defense, if the bot puts the data in, the moment the job is created, on our side we need to be able to know when it is an embargoed update, and not ask per test if it is. As it makes yet another case for dependencies outside of QE.

But at least for now, this is secondary.